index.html

```diff
@@ -117,6 +117,25 @@ <h2>Code</h2>
     level whenever inappropriate behaviors are observed.</li>
 </ol>
 </section>
+	  
+<section id="Reporting">
+	<h2>Reporting</h2>
+	<p>Violations of the code of conduct should be reported to a W3C Ombudsperson as outlined in the complementary <a href="https://www.w3.org/Consortium/pwe/#Procedures">Procedures</a> document.</p>
+	  </section>
+	  <section id="mistake">
+	<h2>If you make a mistake</h2>
+	<p>As we engage in diverse communities we may accidentally cause offense, whether through using unknowingly offensive terminology or through missing social cues.</p>
+<p>If you realise (or are told) that you have offended someone then take the appropriate steps:
+<ol>
+	<li>Accept that you have made a mistake</li>
+	<li>Briefly apologize. Don't try to explain yourself or minimise the issue.</li>
+<li>If possible, edit your message, restate your communication in a better way or withdraw your statement. Publicly revising your statement helps define the culture for others.</li></ul>
+
+	<p>Alice: “Yeah I used X and it was really crazy!” Eve: “Hey, could you not use that word? What about ‘ridiculous’ instead?” Alice: “oh sorry, sure.” -> edits old message to say “Yeah I used X and it was really confusing!”</p>
+<p>This will allow conversation to quickly continue without any need of further action or escalating the situation.</p>
+<p>If you don't understand what you did wrong, assume the the hurt party has good cause and accept it. We cannot know everyone's background and should do our best to avoid harm. You are welcome to discuss it with the ombuddy later.</p>
```

Comment:
s/ombuddy/Ombudperson 

I'm pretty sure we discussed the term "ombuddy" but there's no issue open for adopting it, and for consistency we should use same term throughout.

Reply:
agreed to use ombudsperson in formal documentation